chef/cookbooks/neutron/templates/default/ovs_neutron_plugin.ini.erb

```diff
@@ -64,7 +64,7 @@
 # the server.  In most cases, the default value should be fine.
 #
 # tunnel_bridge = br-tun
-<% if @ml2_type_drivers.include?("gre") -%>
+<% if @ml2_type_drivers.include?("gre") || @ml2_type_drivers.include?("vxlan") -%>
```

Comment:
Any other opinion on this? I find the current code more readable, but I'm happy to change if others agree with @aspiers that's it's cleaner the other way.